src/gui/tray/SyncStatus.qml

```diff
@@ -0,0 +1,115 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import Style 1.0
+
+import com.nextcloud.desktopclient 1.0 as NC
+
+Item {
+    id: item
+
+    property var model: NC.SyncStatusModel {}
+
+    implicitHeight: layout.height
+
+    RowLayout {
+        id: layout
+
+        Layout.alignment: Qt.AlignLeft
+
+        width: item.width
+        spacing: 0
+
+        Image {
+            id: syncIcon
+
+            Layout.alignment: Qt.AlignLeft | Qt.AlignVCenter
+            Layout.topMargin: 16
+            Layout.bottomMargin: 16
+            Layout.leftMargin: 16
+
+            source: model.syncIcon
+            sourceSize.width: 32
+            sourceSize.height: 32
+        }
+
+        Item {
+            id: containerItem
+
+            Layout.alignment: Qt.AlignVCenter
+            Layout.leftMargin: 10
+            Layout.fillWidth: true
+            Layout.fillHeight: true
+
+            ColumnLayout {
```

Comment:
let's merge the two `ColumnLayout` in only one that can show or hide the `ProgressBar`

Reply:
@mgallien I remember why I did it this way. Even if the progress bar is not visible it will occupy space in the `ColumnLayout`